src/lsp/sql_parser/lexer.mll

```diff
@@ -0,0 +1,170 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+{
+  open Grammar
+
+  (*In alphabetical order*)
+  let get_keyword x = 
+  match String.uppercase_ascii x with
```

Comment:
I think here a lookup in a pre-computed hash-table would be preferable. Can be for a later PR.